platooning_strategic/src/platoon_strategic.cpp

```diff
@@ -129,6 +131,36 @@ namespace platoon_strategic
         }
     }
 
+    void PlatoonStrategicPlugin::checkForRightMostLane(const lanelet::BasicPoint2d& current_location)
+    {
+        auto current_lanelet = wm_->getLaneletsFromPoint(current_location, 1);
```

Comment:
I'd recommend checking for size on this output, the getLaneletsFromPoint method is not guaranteed to return a non-empty list.

Reply:
added